frame/referenda/src/lib.rs

```diff
@@ -478,6 +492,36 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Refund the Submission Deposit for a closed referendum back to the depositor.
+		///
+		/// - `origin`: must be `Signed` or `Root`.
+		/// - `index`: The index of a closed referendum whose Submission Deposit has not yet been
+		///   refunded.
+		///
+		/// Emits `SubmissionDepositRefunded`.
+		#[pallet::weight(T::WeightInfo::refund_submission_deposit())]
+		pub fn refund_submission_deposit(
```

Comment:
Or we can have one `refund_deposits` instead `refund_decision_deposit` and `refund_submission_deposit`.
Not sure if its fine to force a caller to pay transaction fees for both unreserve.